scripts/download_technique_animations.py

```diff
@@ -8,67 +8,126 @@
 If the file names have changed from the wiki, the technique
 json files will need to be updated as well.
 """
-from urllib.parse import urljoin
 import os.path
+import pathlib
+import tempfile
+from urllib.parse import urljoin
 
 import requests
 from PIL import Image
 from lxml import html
 
 FRAME_SIZE = 64
-WIKI_URL = "https://wiki.tuxemon.org/index.php?title=Category:Technique_Animation"
+WIKI_URL = "https://wiki.tuxemon.org"
+
+TUXEMON_ROOT_DIR = pathlib.Path(__file__).resolve().parent.parent
+ANIMATION_DIR = TUXEMON_ROOT_DIR.joinpath(pathlib.Path("mods/tuxemon/animations/technique"))
+CREDITS_TEMPLATE = '* ["{animation_name}"]({animation_url}) {credits_text}'
+CREDITS_FILENAME = "TECHNIQUE_ANIMATION_CREDITS.md"
```

Comment:
The filename here is a little too long to my taste, but we can change it later when we split CREDITS.md.